--- I:\sla_dev2\usbwidget-ben\src\.saved\USBHelpers-20130318-0047.cpv	2013-03-18 00:47:56.000000000 -0700
+++ I:\sla_dev2\usbwidget-ben\src\USBHelpers.cpv	2013-03-18 12:30:18.000000000 -0700
@@ -1445,11 +1445,11 @@
     if (found) {
         lbuf[strlen(lbuf)-1] = '\0';    // Remove nl-char
         p = strchr(lbuf, '=');          // Scan for '=' separating ven:id from vendor-string
-        p++;                            // Advance to start of vendor-string
+        // p++;                            // Advance to start of vendor-string // 2013-03-18 SHL
 
         /* '=' found then copy remaining string, otherwise null for not-found */
         if (p) {
-            strcpy(buf, p);
+            strcpy(buf, p + 1);         // Copy vendor-string which follows =
             p = buf;
         }
         else {
